### PR TITLE
Abstract out validating user inputs for processing classes

### LINKED ISSUE #101: Abstract out validating user inputs for processing classes

### PR DESCRIPTION
**Pull request by LWachtel1**
Partially fixes #101 and #88
Created a number of Validator classes (contained in byteceps.validators package); these contain methods that validate input for their corresponding methods in Manager classes.

However, not all Manager class methods have corresponding Validator methods; either there was no input to validate, or I was occasionally afraid to refactor due to fear of introducing bugs. I believe this leaves room for further refactoring.

I had to merge @pqienso's PR changes halfway through. However, the merge missed out some of his .toLowerCase() additions. This led to test failures. I believe I have managed to remedy this by manually finding the missing .toLowerCase() and now all tests pass:

image
image

### CODE REVIEW

**Comment by LWachtel1** [comment]
image

File: src/main/java/byteceps/processing/ExerciseManager.java [hunk 1]
```diff
@@ -6,13 +6,20 @@
...
  * Manages operations related to exercises, such as adding, deleting, editing, listing, and searching exercises.
  */
 public class ExerciseManager extends ActivityManager {
-    //@@author V4vern
 
+
+
+    public ExerciseManager() {
+
+    }
```
Comment (joshualeejunyi) [inline src/main/java/byteceps/processing/ExerciseManager.java:20]: Is there a reason for this?

File: src/main/java/byteceps/validators/HelpValidator.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import byteceps.ui.strings.HelpStrings;
+import byteceps.commands.Parser;
+
+
+//@@author LWachtel1
+public class HelpValidator {
+
+
+    //@@author LWachtel1
+    //@@author joshualeejunyi
```
Comment (joshualeejunyi) [inline src/main/java/byteceps/validators/HelpValidator.java:14]: I don't think this works? Reposense attributes the author until the next author tag if I'm not wrong.

File: src/main/java/byteceps/validators/WorkoutValidator.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+import byteceps.activities.Workout;
+import byteceps.commands.Parser;
+import byteceps.errors.Exceptions;
+import byteceps.ui.strings.CommandStrings;
+import byteceps.ui.strings.ManagerStrings;
+
+
+public class WorkoutValidator {
+
+
```
Comment (joshualeejunyi) [inline src/main/java/byteceps/validators/WorkoutValidator.java:14]: Also can we reduce all the whitespace?

**Review by joshualeejunyi — COMMENTED** [review]
Looks fine, can merge but take note of the comments

**Review by joshualeejunyi — APPROVED** [review]

**Pull request merged**